### PR TITLE
access-control: add test to verify that no SSH daemons are run in a pod

### PR DESCRIPTION
**Pull request by jmontesi**

### CODE REVIEW

File: cnf-certification-test/identifiers/identifiers.go [hunk 1]
```diff
@@ -997,4 +1001,12 @@ that there are no changes to the following directories:
 		BestPracticeReference: bestPracticeDocV1dot3URL + " Section 10.6",
 		ExceptionProcess:      NoDocumentedProcess,
 	},
+	TestNoSSHDaemonsAllowedIdentifier: {
+		Identifier:            TestNoSSHDaemonsAllowedIdentifier,
+		Type:                  normativeResult,
+		Description:           formDescription(TestNoSSHDaemonsAllowedIdentifier, `Check that pods do not SSH daemons.`),
```
Comment (edcdavid) [inline cnf-certification-test/identifiers/identifiers.go:1007]: Check that pods do not SSH daemons. -> Check that pods do not run SSH daemons.

File: cnf-certification-test/identifiers/remediation.go [hunk 2]
```diff
@@ -179,4 +179,6 @@ const (
 	PodTolerationBypassRemediation = `Do not allow pods to bypass the NoExecute, PreferNoSchedule, or NoSchedule tolerations that are default applied by Kubernetes.`
 
 	PersistentVolumeReclaimPolicyRemediation = `Ensure that all persistent volumes are using the reclaim policy: delete`
+
+	NoSSHDaemonsAllowedRemediation = `Ensure that no SSH daemons are run inside a pod`
```
Comment (edcdavid) [inline cnf-certification-test/identifiers/remediation.go:183]: run->running

**Review by edcdavid — APPROVED** [review]

**Review by sebrandon1 — APPROVED** [review]
Approved pending comments from @edcdavid

File: cnf-certification-test/identifiers/identifiers.go (hunk 1 above)
Comment (jmontesi) [reply cnf-certification-test/identifiers/identifiers.go:1007]: Done.

File: cnf-certification-test/identifiers/remediation.go (hunk 2 above)
Comment (jmontesi) [reply cnf-certification-test/identifiers/remediation.go:183]: Done.

File: CATALOG.md [hunk 3]
```diff
@@ -228,6 +228,19 @@ Result Type|normative
...
+
+Property|Description
+---|---
+Test Case Name|ssh-daemons
+Test Case Label|access-control-ssh-daemons
+Unique ID|http://test-network-function.com/testcases/access-control/ssh-daemons
+Version|v1.0.0
+Description|http://test-network-function.com/testcases/access-control/ssh-daemons Check that pods do not SSH daemons.
+Result Type|normative
+Suggested Remediation|Ensure that no SSH daemons are run inside a pod
```
[the remark this answers is not in the document]
Comment (jmontesi) [reply CATALOG.md:241]: Done.

**Pull request merged**